### PR TITLE
Bug/826/differentiate take 2

### PR DESCRIPTION
**Pull request by steventlamb**

### CODE REVIEW

**Comment by maurizi** [comment]
The "No, restart" link works correctly.

The "No, Submit for Review" and "Yes" links still give me a popup that says "You've mapped all your reserved blocks!"

**Comment by steventlamb** [comment]
Oh, are you saying that when you complete the survey from an event, you get the reserved blocks issue, which is not connected to event surveys?

**Comment by maurizi** [comment]
@steventlamb Yes.

**Comment by steventlamb** [comment]
@jwalgran @maurizi updated

**Comment by jwalgran** [comment]
Testing this now

File: src/nyc_trees/js/src/surveyPage.js [hunk 1]
```diff
@@ -458,8 +458,12 @@ function submitSurveyWithTrees() {
             data: JSON.stringify(data)
         })
         .done(function(content) {
+            // make sure not to take the hashstring along for the ride
+            var href = window.location.origin +
+                    window.location.pathname +
+                    'confirm/' + content.survey_id + '/';
```
Comment (jwalgran) [inline src/nyc_trees/js/src/surveyPage.js]: Whenever we embed paths in JS, we have been leaving comments for future refactorers in the js and py files that say something like "this URL needs to be kept in sync with such/and/such/view".

**Comment by jwalgran** [comment]
+1 event mapping and free mapping both "loop around" to the correct pages when a survey is finished.

**Pull request merged**